src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -71,6 +75,66 @@ public void RunBenchmarks(string assemblyPath)
             s_profiler(assemblyPath, Configuration.RunId, _outputDirectory, runner, collectOutputFilesCallback);
         }
 
+        public void RunStartupBenchmark(string[] args, ProcessStartInfo psi, PreDel pre, MidDel mid, PostDel post)
+        {
+            pre(_performanceTestConfig, psi);
```

Comment:
I agree that I don't think we want to run setup every time (we should at least have a way to do once-only setup for big tasks, like building something). Either way, we want to make sure to document this well.